src/sinks/gcp/stackdriver_logs.rs

```diff
@@ -159,6 +160,13 @@ impl HttpSink for StackdriverSink {
     type Output = Vec<BoxedRawValue>;
 
     fn encode_event(&self, event: Event) -> Option<EncodedEvent<Self::Input>> {
+        let mut labels = HashMap::with_capacity(self.config.resource.labels.len());
+        for (key, template) in &self.config.resource.labels {
+            if let Ok(value) = template.render_string(&event) {
```

Comment:
Could we emit an event for the error case? Example:

https://github.com/timberio/vector/blob/30baaeb6bea77c37d60d34ee56860329e7e347f5/src/sinks/aws_s3.rs#L406-L410